cardano_node_tests/utils/dbsync_utils.py

```diff
@@ -474,7 +474,7 @@ def get_pool_data(pool_id_bech32: str) -> PoolDataRecord:
     single_host_names = []
     single_host_addresses = []
 
-    latest_registered_tx_id = max(pool.registered_tx_id for pool in pools)
+    latest_registered_tx_id = pools[-1].registered_tx_id
```

Comment:
List of records returned by `query_pool_data` is already sorted by `registered_tx_id`, so the last record has the highest value in `registered_tx_id`.